ci/setup-build-test-gdk.ps1

```diff
@@ -82,7 +82,9 @@ if ((Test-Path env:TEST_CONFIG) -And ($env:TEST_CONFIG -eq "Native")) {
 else {
     # We run all tests and networked functional maps
     $tests += [TestSuite]::new($gdk_test_project, "SpatialNetworkingMap", "TestResults", "SpatialGDK.+/Game/Maps/FunctionalTests/CI_Fast/", "$user_gdk_settings", $True, "$user_cmd_line_args")
-    
+    $tests += [TestSuite]::new($gdk_test_project, "SpatialNetworkingMap", "SpatialOnlyTestResults", "SpatialGDK.+/Game/Maps/FunctionalTests/CI_Fast_Spatial_Only/", "$user_gdk_settings", $True, "$user_cmd_line_args")
```

Comment:
Actuall now that I am thinking about it, this would be better to collapse into the one above by doing 
```
$tests += [TestSuite]::new($gdk_test_project, "SpatialNetworkingMap", "TestResults", "SpatialGDK.+/Game/Maps/FunctionalTests/CI_Fast/+/Game/Maps/FunctionalTests/CI_Fast_Spatial_Only/", "$user_gdk_settings", $True, "$user_cmd_line_args")
```
That will cause it to run all the tests in one process, rather than spinning up a new one.